.github/contributing.md

```diff
@@ -0,0 +1,213 @@
+
+# 👩‍💻 Contributor's Guide
+
+## 🔁 Software Development Life Cycle
+This project uses [Github Flow](https://docs.github.com/en/get-started/using-github/github-flow)
+for its software development lifecycle. Github workflow is a simple workflow 
+that works well for a small project with multiple developers needing to work 
+on the code and review each other's code before merging into the main branch.
+
+### Process
+
+1. **Developer A** wants to make a change to the code
+2. On their local machine, developer A checkouts a branch to make that change
+3. **Developer B** wants to make a change to a different part of the code
+4. On their local machine, developer B checkouts a branch to make their change
+5. **Developer A** makes the change and pushes the code to the remote repo on Github
+6. **Developer A** opens a Pull Request (PR) so that others can review the change
+7. Other developers approve the Pull Request 
+8. **Developer A** squashes their branch's commits and merges the Pull Request into the main branch
+9. **Developer B** updates their local main branch with the changes that were just merged 
+10. **Developer B** squashes their branch's commits and merges the Pull Request into the main branch
+11. **Developer B** opens a Pull Request (PR) so that others can review the change
+12. Other developers approve the Pull Request 
+13. **Developer C** wants to make a release
+14. **Developer C** uses the [D3b Release Maker](https://github.com/d3b-center/d3b-release-maker) to 
+create a release PR on Github
+15. Other developers approve the Pull Request 
+16. **Developer C** merges the release PR which activates a Github workflow that tags the main branch with 
+the next version and creates the release on Github 
+
+
+Let's solidify this with an example.
+
+## 👨🏻‍💻 Develop
+
+This assumes you've already setup your development environment. If not, 
+follow the steps in [Setup Dev Environment](#setup-dev-environment)
+
+### Create a branch
+```shell
+# Checkout your local main branch
+git checkout main
+
+# Update your local main branch to mirror the main branch on Github
+git pull
+
+# Create a feature branch to work on. Name it something short and relevant 
+# to the change
+git checkout -b add-patient-birth-sex 
+```
+
+### Commit the change 
+
+After you've made the change you want on your local machine its time 
+to commit that change to the project repo. 
+
+#### Emoji Commits
```

Comment:
Update wording to indicate that the gitmoji's are optional or at least strict adherence won't be required. Will still be included to give more robust search options for browsing commit history.